inlong-dashboard/src/plugins/sources/common/SourceDefaultInfo.ts

```diff
@@ -136,6 +136,10 @@ export class SourceDefaultInfo implements DataWithBackend, RenderRow, RenderList
   @I18n('basic.Status')
   readonly status: string;
 
+  @ColumnDecorator()
+  @I18n('meta.Sources.ClusterName')
+  readonly inlongClusterName: string;
```

Comment:
The `inlongClusterName` should be placed before the `status`.

Reply:
done